distribution/changelog.txt

```diff
@@ -39,6 +41,7 @@
 - Fix: [#13222] Vehicle collision causes negative number of passengers (original bug).
 - Fix: [#13226, #7280] No error is shown when attempting to load a corrupted save.
 - Fix: [#13266] Plugin API: Deleting key of sharedStorage not working.
+- Fix: [#13278] Desync caused by to ghost tiles changing the ride mode.
```

Comment:
This seems to be a typo.

Reply:
Fixed in https://github.com/OpenRCT2/OpenRCT2/blob/develop/distribution/changelog.txt, thanks!